app/utils/sentry.ts

```diff
@@ -39,9 +40,19 @@ export function initializeSentry() {
         return;
     }
 
+    const mmConfig = {
+        environment: isBetaApp ? 'beta' : 'production',
+        tracesSampleRate: isBetaApp ? 1.0 : 0.2,
+        sampleRate: isBetaApp ? 1.0 : 0.2,
+        attachStacktrace: isBetaApp, // For Beta, stack traces are automatically attached to all messages logged
+        logLevel: isBetaApp ? Sentry.LogLevel.Debug : Sentry.LogLevel.Error,
```

Comment:
is LogLevel.Error only recording fatal errors?

Reply:
I believe so, other values would be too noisy for production

Reply:
I believe so is not really an answer hahahaha, I think you should check for is fatal in the capture JS exception function